contracts/MultichainProxy.sol

```diff
@@ -282,10 +470,15 @@ contract MultichainProxy is OnlySourceFunctionality {
         address _underlyingToken
     ) private {
         // Give Anyswap approval to bridge tokens
-        SmartApprove.smartApprove(_underlyingToken, _amount, _anyRouter);
+        SmartApprove.smartApprove(_underlyingToken, _amount, _anyRouter); // TODO test approves only on underlying
```

Comment:
TODO

Reply:
I'm not sure that this will work the same way as it worked earlier with all kinds of tokens, and it very hard to test. Will leave everything as it is. 
https://github.com/Cryptorubic/Multichain-proxy/commit/7c28f41b661a302e87d6a4c3e0ee9fa3d5394022